server/integration-tests/data-survives-crash.sh

```diff
@@ -7,11 +7,17 @@ fi
 
 ########## Preparations
 stack build
-DATA_FILE=`mktemp`
-echo '{}' > "$DATA_FILE"
+
+BACKEND_FLAGS=("--file" "--sqlite")
 
 ########## 1st Pass
-stack exec -- icepeak --data-file="$DATA_FILE" --sync-interval 60s --journaling > /dev/null &
+
+for item in ${BACKEND_FLAGS[*]}
+do
+echo ">>>>>>>> Testing backend: $item"
```

Comment:
This looks a lot like a Git conflict marker, it would be nice to use something else, to avoid confusion and also to search for the conflict marker when there is a conflict without hitting this false positive.

Reply:
True, fixed.

Reply:
This has not been fixed?

Reply:
I had fixed it in `data-survives-restart.sh`, but then forgotten to fix it here. Now it's fixed.